eth/backend.go

```diff
@@ -42,18 +43,20 @@ type Backend interface {
 type backend struct {
 	*ethclient.Client
 	methods      map[string]string
+	abiMap       map[string]abi.ABI
```

Comment:
should we turn this into a pointer ref?

Reply:
Yeah seems reasonable for avoiding allocating multiple ABI instances.

Reply:
fixed in 8a965a6